packages/frontend/src/contexts/flagsContext.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo, useState } from 'react'
+
+import flags from '../utils/flags'
+
+const DEFAULT_FLAGS_STATE = {}
+
+const FlagContext = React.createContext(DEFAULT_FLAGS_STATE)
+
+export function useFlag() {
```

Comment:
Hmm so, I think the name of the function is a tad misleading. You're not accessing the flag, but rather the context that holds the flag. Maybe we should call it `useFlags`?
```suggestion
export function useFlags() {
```